content/en/blog/_posts/2023-03-01-introducing-kwok/index.md

```diff
@@ -0,0 +1,78 @@
+
+- **Speed**: You can create and delete clusters and nodes almost instantly, without waiting for boot or provisioning.
+- **Compatibility**: KWOK works with any tools or clients that are compliant with Kubernetes APIs, such as kubectl, helm, kui, etc.
+- **Portability**: KWOK has no special hardware or software requirements. You can run it on your laptop, desktop or any cloud instance that supports Docker, Nerdctl, or even without container using binaries, but in non-Linux you need to build the control plane on the source. You can also run it alongside other Kubernetes clusters or tools.
+- **Flexibility**: 
+  - You can simulate thousands of nodes on a single machine without great consumption of CPU or memory resources.
+  - You can configure different node types, labels, taints, capacities, conditions, etc., and you can configure different pod behaviors, status, etc. to test different scenarios and edge cases.
+
+## What are the use cases?
+
+KWOK can be used for various purposes:
+
+- **Learning**: You can use KWOK to learn about Kubernetes concepts and features without worrying about resource waste or other consequences. 
+- **Development**: You can use KWOK to develop new features or tools for Kubernetes without accessing to a real cluster or requiring other components.
+- **Testing**:
+  - You can measure how well your application or controller scales with different numbers of nodes or pods.
+  - You can generate high loads on your cluster by creating many pods or services with different resource requests or limits.
+  - You can simulate node failures or network partitions by changing node conditions or randomly deleting nodes.
+  - You can test how your controller interacts with other components or features of Kubernetes by enabling different feature gates or API versions.
+
+## What are the limitations?
+
+KWOK is not intended to replace others completely. It has some limitations that you should be aware of:
+
+- **Functionality**: KWOK is not a kubelet and many behaviors may be different, such as pod lifecycle management, volume mounting, device plugins, etc. It only simulates updates of node status and pod status.
+- **Accuracy**: KWOK does not reflect the actual performance or behavior of real nodes under different workloads or environments. It only approximates some behaviors based on simple formulas.
+- **Security**: KWOK does not enforce any security policies or mechanisms on simulated nodes. It assumes that all requests from the kube-apiserver are authorized and valid.
+
+## Getting started
+
+If you are interested in trying out KWOK, please check its [documents] for more details.
+
+{{< figure src="/blog/2023/03/01/introducing-kwok/manage-clusters.svg" alt="Animation of a terminal showing kwokctl in use" caption="Using kwokctl to manage simulated clusters" >}}
+
+## Getting Involved
+
+If you're interested in future development and discussions on KWOK, you can get involved in SIG Scheduling through the following ways:
```

Comment:
Again, don't need to mention scheduling, kwok is aimed for general use scenarios, not just scheduling

```suggestion
If you're interested in participating in future discussions or development related to KWOK, there are several ways to get involved:
```